src/index.js

```diff
@@ -277,41 +283,64 @@ async function manageBoard (reaction) {
       })
 
       // post embed
-      postChannel.send({ embeds: embeds }).then(starMessage => {
+      let starMessage
+      try {
+        if (webhook) {
+          starMessage = await webhook.send({
+            username: client.user.username,
+            avatarURL: client.user.avatarURL(),
```

Comment:
it might be interesting to set the webhook username and avatar to that of the person who authored the starred message

optimally as a toggleable feature

Reply:
sounds good to me, I'll add it as a toggleable feature in the config

Reply:
I think it's a good idea, but I think if I add this it should remove the title prop where the name also resides. Looks pretty off if it has both

![image](https://github.com/naeruru/starboard/assets/9059594/4042987f-7ef2-46b9-9882-49523e10836b)

Reply:
yeah i agree, that's how I imagined it! just remove the author field from the embed if using webhook

____

also one last thing to consider, webhook usernames may not contain the following terms:
- Discord
- System Message
- Clyde
- Jason Citron's password

but user usernames also may not contain these  (nicknames / displaynames can), so as long as it only uses usernames, it's fine

Reply:
Ah okay, sounds good!!